point_of_sale/belgian_fdm/setup.rst

```diff
@@ -0,0 +1,82 @@
+===================================================
+Setting up the Fiscal Data Module with the Odoo POS
+===================================================
+
+Introduction
+============
+
+The Belgian government requires certain businesses to use a
+government-certified device called a Fiscal Data Module (also known as
+a blackbox). This device works together with the POS application and
+logs certain transactions. On top of that, the used POS application
+must also be certified by the government and must adhere to strict
+standards specified by them. `Odoo 9 is a certified application
+<http://www.systemedecaisseenregistreuse.be/systemes-certifies>`_. More
+information concerning the Fiscal Data Module can be found on `the
+official website <http://www.systemedecaisseenregistreuse.be/>`_.
+
+**Currently our government certificate is only valid for Odoo
```

Comment:
probably add a date on that statement, in case somebody keeps a copy of an old doc somewhere